specs/eventing/data-plane.md

```diff
@@ -169,12 +166,14 @@ event recipient that the caller will accept a
 [`200` response](#event-acknowledgement-and-repeat-delivery) which includes a
 CloudEvent encoded using the binary or structured formats.
 [Brokers](./overview.md#broker) and [Channels](./overview.md#channel) MUST
-indicate support for replies using the `Prefer: reply` header.
+indicate support for replies using the `Prefer: reply` header when sending to
+the `spec.subscriber` address.
 
 A recipient MAY reply to any HTTP POST with a `200` response to indicate that
 the event was processed successfully, with or without a response payload. If the
 recipient will _never_ provide a response payload, the `202` response code is
-likely a better choice.
+also acceptable. Responses with a `202` response code MUST NOT be processed as
```

Comment:
Can we make this a bit stronger? Something like:
...as reply events since a `202` (even if there is a body) does not indicate successful processing of the event - only that it was received.

Reply:
I'm not sure there's a stronger than "MUST NOT", but I can add some rationale. From RFC 7231:

>    The representation sent with this
>    response ought to describe the request's current status and point to
>    (or embed) a status monitor that can provide the user with an
>    estimate of when the request will be fulfilled.

I could imagine a status monitor which looked like a "request enqueued" CloudEvent; this is explicitly ruling out processing that response for a 202.

Reply:
Yes it's not so much about making it more than a MUST NOT, it's more explaining that anything that MIGHT flow in the response IS NOT an indication of successful processing of the event, just successful delivery -and yes, as you said, not a "reply event". I think this will be lost by a novice reader.